packages/components/pie-checkbox/src/checkbox.scss

```
@@ -1 +1,275 @@
 @use '@justeattakeaway/pie-css/scss' as p;
+
+@keyframes checkboxCheck {
+    0% {
+        width: 0;
+        height: 0;
+        border-color: #fff;
+
+        transform: translate3d(0, 0, 0) rotate(45deg);
+    }
+
+    33% {
+        width: 8px;
+        height: 0;
+
+        transform: translate3d(0, 0, 0) rotate(45deg);
+    }
+
+    100% {
+        width: 8px;
+        height: 16px;
+        border-color: #fff;
+        border-bottom-right-radius: 2px;
+
+
+        transform: translate3d(0, -16px, 0) rotate(45deg);
+    }
+}
+
+@keyframes scaleDown {
+    0% {
+        transform: scale(0.7);
+    }
+
+    100% {
+        transform: scale(0);
+    }
+}
+
+@keyframes scaleUp {
+    0% {
+        transform: scale(0);
+    }
+
+    33% {
+        transform: scale(0.5);
+    }
+
+    100% {
+        transform: scale(1);
+    }
+}
+
+.c-checkbox {
+    --checkbox-height: 24px;
+    --checkbox-width: 24px;
+    --checkbox-radius: var(--dt-radius-rounded-a);
+    --checkbox-margin: 1px;
+    --checkbox-gap: var(--dt-spacing-b);
+    --checkbox-fz: var(--dt-font-body-l-size);
+    --checkbox-fw: var(--dt-font-weight-regular);
```

I wouldn't use shorthand for these – makes it harder to understand:

```suggestion
    --checkbox-font-size: var(--dt-font-body-l-size);
    --checkbox-font-weight: var(--dt-font-weight-regular);
```